packages/cursorless-engine/src/languages/TreeSitterQuery/rewriteStartOfEndOf.test.ts

```diff
@@ -0,0 +1,61 @@
+import { Range } from "@cursorless/common";
+import { MutableQueryCapture } from "./QueryCapture";
+import { SyntaxNode } from "web-tree-sitter";
+import { rewriteStartOfEndOf } from "./rewriteStartOfEndOf";
+import assert = require("assert");
+
+type NameRange = Omit<MutableQueryCapture, "allowMultiple" | "node">;
```

Comment:
the name `NameRange` implies to me that you care that you have `name` and `range`, so I'd argue to use `Pick` rather than `Omit`, eg
```suggestion
type NameRange = Pick<MutableQueryCapture, "name" | "range">;
```

Reply:
TIL about pick. Thanks. done